services/TokenUpdate.proto

```diff
@@ -53,4 +54,6 @@ message TokenUpdateTransactionBody {
     Duration autoRenewPeriod = 11; // The new interval at which the auto-renew account will be charged to extend the token's expiry.
     Timestamp expiry = 12; // The new expiry time of the token. Expiry can be updated even if admin key is not set. If the provided expiry is earlier than the current token expiry, transaction wil resolve to INVALID_EXPIRATION_TIME
     google.protobuf.StringValue memo = 13; // If set, the new memo to be associated with the token (UTF-8 encoding max 100 bytes)
+    CustomFees customFees = 14; // Custom fee in Token Units/hbar that needs to be paid to the specified receiver on a TokenTransfer
```

Comment:
Can a custom fee be deleted or updated, or only created, and if so how will that look in this object?

Reply:
@tinker-michaelj I think this question got passed over, still relevant.

Reply:
I do see the comments for updating the fees for a token , to clarify can one actually update a particular fee, or is it just "on an update, all existing fees are cleared and the new fees replace them."